api/api_test.go

```diff
@@ -491,11 +575,11 @@ func launchServer(t *testing.T) func() {
 	return func() {
 		jsonService.StopService()
 		grpcService.StopService()
-	}
+	}, nodeConf
```

Comment:
why do you need to return this if it's always the default config from your function?

Reply:
do didn't address this comment